docs/docs/cmd/pp/managementapp/managementapp-add.md

```diff
@@ -0,0 +1,50 @@
+# pp managementapp add
+
+Register management application for Power Platform
+
+## Usage
+
+```sh
+m365 pp managementapp add [options]
+```
+
+## Options
+
+`--appId [appId]`
+: Application (client) ID of the Azure AD application registration to register as management app. Specify either `appId`, `objectId` or `name`
+
+`--objectId [objectId]`
+: Object ID of the Azure AD application registration to register as management app. Specify either `appId`, `objectId` or `name`
+
+`--name [name]`
+: Name of the Azure AD application registration to register as management app.. Specify either `appId`, `objectId` or `name`
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+To execute this command the first time you will need to be signed in using the default Azure PowerShell App Registration `1950a258-227b-4e31-a9cf-717495945fc2`. Run `m365 login --appId 1950a258-227b-4e31-a9cf-717495945fc2` before executing this command. If you are using your the default CLI for Microsoft 365 you can use `--appId 31359c7f-bd7e-475c-86db-fdb8c937548e` to register.
+
+For best performance use the `appId` option to reference the Azure AD application registration to update. If you use `objectId` or `name`, this command will first need to find the corresponding object ID for that application.
```

Comment:
In the last part of the sentence, let's change `corresponding object ID for that application` to `corresponding appId for that application` as that's what we're after